stories/DatePicker.js

```diff
@@ -1,19 +1,29 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
```

Comment:
It is deprecated import of `PropTypes`, you should take them from `import PropsTypes from 'prop-types';`